do_configure.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
This seems to be a private, specific script and I question it's general use and thus the addition to the repo